primitives/src/market.rs

```diff
@@ -112,6 +112,7 @@ pub enum MarketDisputeMechanism<AI> {
     Authorized(AI),
     Court,
     SimpleDisputes,
+    GlobalDisputes,
```

Comment:
Smells a little bit like storage migraton, but on the other hand an enum value at the end conflicts not with the past values. So maybe no migration required?